menas/ui/service/MonitoringService.js

```diff
@@ -296,8 +300,12 @@ var MonitoringService = new function() {
   };
 
   // find raw recordcount
-  // TODO: Compare controlls across all checkpoints
-  this.processCheckpoints = function(oRun) {
+  this.setRawRecordcount = function (oRun) {
+    let rawRecordcount = oRun["controlMeasure"]["metadata"]["additionalInfo"]["raw_record_count"];
+    if (rawRecordcount != null && !isNaN(rawRecordcount) ) {
+      oRun["rawRecordcount"] = rawRecordcount;
+      return;
```

Comment:
What is the point of this `return`?

Reply:
If the raw_record_count is provided in run metadata (as in the new Enceladus version) we do not need to search it in the checkpoints, therefore, we exit the method here.